### DISCUSSION TITLE
Standalone equation in Typst?

### DISCUSSION THREAD

**Discussion by Kreijstal** [typst/typst#893] [Q&A] [answered]
How can I create a "standalone" PDF? basically a PDF that is as big as the contents?
Like when we can do standalone TiKz, or equations?

I mean you can do something like
```
#set page(
  width: 3cm,
  height:50pt,
  margin: (x: 0cm),
)
```
But do I have to hardcode the width to my diagram or svg or something like that?

**Comment by laurmaedje** [comment, marked as answer]
You can set width and height to `auto`.
